packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -153,6 +154,7 @@ export function addRoutesToApp(app) {
   app.post('(/v[0-9]+)/import/users', upload.single('users'), importUsers);
   app.post('(/v[0-9]+)/import/optionSets', upload.single('optionSets'), importOptionSets);
   app.post('(/v[0-9]+)?/user', createUser);
+  app.post('(/v[0-9]+)?/userAccount', createUserAccount);
```

Comment:
Might be nice to have a comment explaining the difference between these end points. Would be confusing if you're reading this without context.